src/util/text-utils.cc

```diff
@@ -571,8 +571,8 @@ void ReadConfigLines(std::istream &is,
     size_t end = line.find_first_of('#');
     if (start == std::string::npos || start == end) continue;
     end = line.find_last_not_of(" \t", end - 1);
-    KALDI_ASSERT(end >= start);
-    lines->push_back(line.substr(start, end - start + 1));
+    if (end > start)
```

Comment:
I don't understand why the case where end == start would be a problem.  There would be a one-character string, no?  (Note, here, "end" is really last, not one past the last).

Reply:
The main reason I made it because address sanitizer was complaining.

Reply:
Ah, I see. Now it skips one-character values. I will revert this change, and will look at it later.

Reply:
I think this was a false alarm, because it can't see the full logic of the routine.  depends on behavior of those string functions

Reply:
I removed this commit from the pull request